packages/gen_js_api/gen_js_api.1.0.3/opam

```diff
@@ -15,4 +15,4 @@ depends: [
   "ocamlfind" {build}
   "js_of_ocaml"
 ]
-available: [ocaml-version >= "4.03.0"]
+available: [ocaml-version >= "4.03.0" & ocaml-version <= "4.05"]
```

Comment:
There are earlier versions in the repo without a similar upper bound -- do they also need a constraint added?